rfc6962-bis.xml

```diff
@@ -462,13 +458,10 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
 
     struct {
         CMSPrecert pre_certificate;
-        ASN.1Cert precertificate_chain&lt;0..2^24-1&gt;;
+        ASN.1Cert precertificate_chain&lt;1..2^24-1&gt;;
```

Comment:
It's >0 because (at least) the root certificate MUST be included in precertificate_chain.  (The root certificate can't be the pre_certificate, and we've said that the root must be stored as part of the entry).

I think 1 is the correct lower bound.  Consider a precertificate for a name-constrained intermediate that will be signed directly by a root - surely the precertificate_chain will contain precisely one certificate (i.e. the root certificate)?